### DISCUSSION TITLE
Search for words in articles feature

### DISCUSSION THREAD

**Discussion by goliv04053** [jocmp/capyreader#296] [Feature Requests]
The idea is: You have a search bar and it fetches the results based on articles you have and it give the best match to it.

**Comment by jocmp** [comment]
thanks @goliv04053 - I'll track this as a part of https://github.com/jocmp/capyreader/issues/91.

**Comment by jocmp** [comment]
Search is available as of [2024.08.1036-dev](https://github.com/jocmp/capyreader/releases/tag/2024.08.1036-dev). It searches in the article's title and summary. Let me know if it's useful. I may switch this to also search within the article content too if it stays performant.
